### ISSUE TITLE
Logged out everytime pc restart

### ISSUE THREAD

**Issue by Hell-riser** [ytmdesktop/ytmdesktop#1227]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

I am having a problem where I restart my pc and come to find the app signed out of google. Has happened multiple times now.

### Expected Behavior

I expected to still be signed in after a pc restart.

### Steps To Reproduce

1. Sign in to Youtube Music Desktop Application.
2. Restart PC.
3. Re-open application.
4. No longer signed in.

### YTMDesktop

v1.13.0

### OS

Windows

### OS Version

10

### Arch

x64

### Installation way

Winget

**Comment by Alipoodle** [comment]
> Also shows in the right click menu in the task bar "guest" even while signed in.
This is a Guest mode which is intended for when you are NOT signed in.

But this is the first time I've heard this happening.
Could you share a screenshot of your YTM For me (including the titlebar), and What the initial bit of the login screen looks like. (don't need to have any details put in?

Many Thanks

**Comment by Hell-riser** [comment]
![image](https://github.com/ytmdesktop/ytmdesktop/assets/154860304/79849d2e-d564-4835-abd0-5f325a6b8c67)
![image](https://github.com/ytmdesktop/ytmdesktop/assets/154860304/2aa6e584-41fe-4782-994b-40a8b9eaf081)
![image](https://github.com/ytmdesktop/ytmdesktop/assets/154860304/4da707b2-1d98-4102-98b8-65aea824b844)
![image](https://github.com/ytmdesktop/ytmdesktop/assets/154860304/aaa9f12b-d267-4ca4-8098-f813759579f2)

**Comment by Hell-riser** [comment]
![image](https://github.com/ytmdesktop/ytmdesktop/assets/154860304/5545992c-d534-437b-bf9a-fe6c749ed05f)

**Comment by Hell-riser** [comment]
![image](https://github.com/ytmdesktop/ytmdesktop/assets/154860304/c949d8fd-ebed-4b93-add8-73b3d34ed8e9)
this shows up every so often too

**Comment by Alipoodle** [comment]
> ![image](https://private-user-images.githubusercontent.com/154860304/293106700-c949d8fd-ebed-4b93-add8-73b3d34ed8e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM3ODk3ODksIm5iZiI6MTcwMzc4OTQ4OSwicGF0aCI6Ii8xNTQ4NjAzMDQvMjkzMTA2NzAwLWM5NDlkOGZkLWViZWQtNGI5My1hZGQ4LTczYjNkMzRlZDhlOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyOFQxODUxMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NTk1ZjRmZDg2YmQ1MGNiNDQ2Zjg3ODk1MjE4YjNmYWVkOTZkYTBiZmJhZThkZDJhZmI5ZjJhNTA4ODIyNGVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-q4t0qdH5jqntInBZuk_w_8XRt0bhpcJGuvrYRPgozc) this shows up every so often too

This is an issue where another application is open using the same 9863 port... OR you have multiple instances of YTM open and it's not correctly handling the socket.

If you don't use the Remote Control tool, I recommend disabling it as this is the feature which is causing it.

**Comment by Hell-riser** [comment]
So I need the remote control tool for stream deck integration, however its only local and plugged in through my pc so it shouldnt be going network. I uninstalled and reinstalled YTMDesktop and attempted multiple methods making sure to install all the binaries and stuff and still im stuck in the guest menu.

**Comment by Alipoodle** [comment]
Is it possible to try and update to YTMD v2 now that we've released it and see if the issue continues there.

**Comment by Hell-riser** [comment]
Yeah seems like it has fixed itself, if anything changes when I restart my system I'll lyk.

**Comment by gotgenes** [comment]
I am experiencing a similar situation with YTMD 2.0.0 on MacOS 14.3 (using the arm64 build). The steps to reproduce are to

1. Sign in

   1. Enter my email
   2. Enter my password
   3. Enter my MFA
   4. select my YouTube account (I have multiple accounts*, as described in #1221)

2. Quit the app
3. Open the app again
4. Observe that I am again signed out, and have to sign in again.

A restart is not necessary, although it will cause the same thing, since restart implies quitting all open applications.

I am wondering if @Hell-riser also has multiple YouTube accounts affiliated with his email address.

*Google can't decide whether to call these ["YouTube Accounts" or "YouTube channels"](https://support.google.com/youtube/answer/3046356?hl=en&co=GENIE.Platform%3DDesktop&sjid=10375276449281530625-NC) but they are the same concept.

**Comment by Hell-riser** [comment]
My issues have seemed to resolved, I do believe I have two emails affiliated with many emails linked to my main email as the back up but I only believe I have one youtube account associated with the email I am using.

**Comment by Alipoodle** [comment]
I personally run with a Standard + Branded account on my YouTube account and I am fine with this.
I have been using my account since before we've added in a lot of the restrictions to make sure you stay on YTM or Google specific domains.

**Comment by FlameChrome** [comment]
> I am experiencing a similar situation with YTMD 2.0.0 on MacOS 14.3 (using the arm64 build). The steps to reproduce are to
> 
>     1. Sign in
>        
>        1. Enter my email
>        2. Enter my password
>        3. Enter my MFA
>        4. select my YouTube account (I have multiple accounts*, as described in [Sign-in stuck on latest builds of v2 on MacOS #1221](https://github.com/ytmdesktop/ytmdesktop/issues/1221))
> 
>     2. Quit the app
> 
>     3. Open the app again
> 
>     4. Observe that I am again signed out, and have to sign in again.
> 
> 
> A restart is not necessary, although it will cause the same thing, since restart implies quitting all open applications.
> 
> I am wondering if @Hell-riser also has multiple YouTube accounts affiliated with his email address.
> 
> *Google can't decide whether to call these ["YouTube Accounts" or "YouTube channels"](https://support.google.com/youtube/answer/3046356?hl=en&co=GENIE.Platform%3DDesktop&sjid=10375276449281530625-NC) but they are the same concept.

just installed the 2.0 update today, and i have restarted the app a couple of times (for different reasons) and I have only one google account and its also logging me out everytime i have to reopen the app. However i am running macos 14.2.1 instead, but also on arm due to having the m1pro chip

**Comment by gotgenes** [comment]
I tried poking through some of the files in ` ~/Library/Application\ Support/YouTube\ Music\ Desktop\ App`, and then through some of the code here, but it's not clear to me yet where YTMD intends to persist the sign-in token from YouTube/Google, and why it is failing to retrieve this information on application startup. This behavior started after RC4 (meaning, I didn't experience this with RC4) and I think after RC5, so it was introduced fairly close to the official 2.0 release.

**Comment by gotgenes** [comment]
>But on Windows I'm not sure what's going on

From [above](https://github.com/ytmdesktop/ytmdesktop/issues/1227#issuecomment-1919704000)
>My issues have seemed to resolved

Do you want to close this issue, and open a separate issue for MacOS, since it seems distinct (and possibly there never was a bug on Windows)?

**Comment by csrednicki** [comment]
> https://github.com/ytmdesktop/ytmdesktop/issues/1227#issuecomment-1919393498

same issue, I need to login every time I start app. After install I had to use this command to even open it, maybe this is related?
```
/usr/bin/xattr -cr /Applications/YouTube\ Music\ Desktop\ App.app
```

MacOS 26.4, app version 2.0.11 3d49f52

**Comment by marleymcbride** [comment]
@csrednicki this is happening for me too. did you find a fix?

**Comment by csrednicki** [comment]
> [@csrednicki](https://github.com/csrednicki) this is happening for me too. did you find a fix?

nope
